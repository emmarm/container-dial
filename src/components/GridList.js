import React from 'react';
import PropTypes from 'prop-types';
import styled from 'react-emotion';

const Grid = styled('div')({
  display: 'grid',
  gridGap: '20px',
  gridTemplateColumns: 'repeat(auto-fit, 220px)',
  justifyContent: 'center',
  margin: '40px'
});

const GridList = ({ children }) => <Grid>{children}</Grid>;

GridList.propTypes = {
  children: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
};

export default GridList;
