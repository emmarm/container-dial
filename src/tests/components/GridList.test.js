import React from 'react';
import { shallow } from 'enzyme';

import GridList from '../../components/GridList';

describe('GridList component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<GridList />);
    expect(wrapper).toMatchSnapshot();
  });
});
