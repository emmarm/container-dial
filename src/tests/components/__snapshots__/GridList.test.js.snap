// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridList component matches snapshot 1`] = `<Styled(div) />`;
